avaframe/runCom1DFA.py

```diff
@@ -33,7 +33,7 @@
 startTime = time.time()
 
 # Clean input directory(ies) of old work and output files
-initProj.cleanSingleAvaDir(avalancheDir, keep=logName)
+initProj.cleanSingleAvaDir(avalancheDir, keep=logName, deleteOutput=False)
```

Comment:
I think we should leave the default option of deleting the output folder as well...